src/componenets/protected-area/service.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from "react"
+
+class Service extends Component<ServiceProps, ServiceState> {
```

Comment:
I see we are using both functional and class-based components, there is absolutely no problem using them, but should we standardize the code?

Reply:
I tried funcional components but I had some issues with types and props, maybe later on I can convert these components to be functional when I understand them more